server/pkg/webserver/controller/gateway.go

```diff
@@ -25,7 +24,6 @@ type connectDBParams struct {
 	Port     int    `json:"port"`
 	Username string `json:"username"`
 	Password string `json:"password"`
-	Version  string `json:"version"`
```

Comment:
API can't specify version anymore?

How about use `Version` as an optional arguments?

Reply:
The caller do not know the version info, so it's need not to add this arguments.